Dispatcher/UI/Profile/Settings/SettingsViewController.swift

```diff
@@ -44,11 +44,17 @@ extension SettingsViewController: UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        
+
         let cell = tableView.dequeueReusableCell(withIdentifier: Constants.TableCellsIdentifier.SETTING, for: indexPath) as! AppSettingCell
-        
+        cell.delegate = self
+        print("======")
```

Comment:
We don't keep this kind of logs in the PR. Please remove unnecessary logs.

Reply:
Fixed
@abenartz